app/models/metric.rb

```diff
@@ -47,4 +47,12 @@ def self.reindex_table_name(time = Time.now.utc.hour + 1)
     hour = (time.respond_to?(:strftime) ? time.hour : time) % 24
     "metrics_%02d" % hour
   end
+
+  def self.metrics_in_range(resource_type, resource_ids, start_date, end_date = nil)
```

Comment:
NOE I called this `metrics_in_range` because the equivalent method on the `MetricRollup` model is `rollups_in_range`

Otherwise I probably would have gone with `Metric.in_range` and `MetricRollup.in_range` since the type is kind of implied by the model you're calling it on